deployments/deployment/nginx-ingress.yaml

```diff
@@ -28,6 +28,13 @@ spec:
           containerPort: 443
        #- name: prometheus
          #containerPort: 9113
+       #- name: readiness-port
```

Comment:
We need to add these changes in `deployments/daemonset/nginx-ingress.yaml` and  `deployments/daemonset/nginx-plus-ingress.yaml` right?